datapackage__emissions.yaml

```diff
@@ -0,0 +1,248 @@
+contributors: "\u017Diga Zaplotnik"
```

Comment:
Sad to see YAML, which is a poor format, but then again I'm not convinced JSON is a better format for this use case, as it does not support comments.